android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1133,6 +1133,11 @@
 	 */
 	public static final String PROPERTY_BUTTON_NAMES = "buttonNames";
 
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_BUTTON_PRESS_REQUIRED = "buttonPressRequired";
```

Comment:
Should we name this `buttonClickRequired` to coincide with the `click` event, to prevent confusion with the `pressed` event?

Reply:
Good point. I'm open to different names. Anyone else have an opinion?